pkg/ddevapp/ddevapp.go

```diff
@@ -1120,6 +1120,7 @@ func (app *DdevApp) DockerEnv() {
 		"DDEV_ROUTER_HTTPS_PORT":        app.RouterHTTPSPort,
 		"DDEV_XDEBUG_ENABLED":           strconv.FormatBool(app.XdebugEnabled),
 		"DDEV_PRIMARY_URL":              app.GetPrimaryURL(),
+		"DDEV_VERSION":                  version.GetVersionInfo(),
```

Comment:
GetVersionInfo() returns a map of lots of stuff, it's not what you want. You just want version.Version.

You can compile things and test them by just using `make` and then running the artifact in .gotmp/bin/windows_amd64/ddev.exe

I *think* that this also has to be added in template.go to the docker-compose template, or the env var won't make it inside the container.

Reply:
Thanks for the hint, fixed now